low-level/sensors/impl/generic/node_hw.py

```diff
@@ -554,6 +612,7 @@ def _notify_NodeDataMsgHandler(self):
 
         if last_index is not None:
             self._write_index_file(last_index)
+        self._check_faulty_resource_status()
```

Comment:
move _check_faulty_resource_status() to line 472 above, as we want to keep it separate from SEL processing and that too execute only after clean SEL processing, which is confirmed after second call to _notify_NodeDataMsgHandler() on line 471

Reply:
Yes agreed, Done.
Addressed this review comments in https://github.com/Seagate/cortx-monitor/pull/393